tiledb/storage_format/uri/test/unit_uri_format.cc

```diff
@@ -180,7 +182,8 @@ TEST_CASE(
           "__1_2_44318efd44f546b18db13edc8d10805b");
 
       // Check timestamp range
-      REQUIRE(get_timestamp_range(frag, &range).ok());
+      FragmentURI fragment_uri{frag};
+      range = fragment_uri.timestamp_range();
       CHECK(range.first == 1);
       CHECK(range.second == 2);
```

Comment:
Can you smell the copypasta? Three tests that only vary by a few parameters. This should be table-driven.